<ui version="4.0" >
 <class>frmDimensoesImagem</class>
 <widget class="QWidget" name="frmDimensoesImagem" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="lblDimsImg" >
     <property name="text" >
      <string>Dimens√µes da imagem:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="lblDimsLarg" >
       <property name="text" >
        <string>&amp;Largura:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>spnbxLarg</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QSpinBox" name="spnbxLarg" >
         <property name="toolTip" >
          <string>Largura da imagem, em pixels.</string>
         </property>
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="maximum" >
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="blDimsLargPix" >
         <property name="text" >
          <string>pixels</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="3" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="lblDimsAlt" >
       <property name="text" >
        <string>&amp;Altura:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>spnbxAlt</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QSpinBox" name="spnbxAlt" >
         <property name="toolTip" >
          <string>Altura da imagem, em pixels.</string>
         </property>
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="maximum" >
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblDimsAltPix" >
         <property name="text" >
          <string>pixels</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="3" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
